docs/examples/bitwarden.md

```
@@ -22,18 +22,24 @@ When you create a new external-secret object, the External Secrets webhook provi
 
 * Bitwarden account (it also works with Vaultwarden!)
 * A Kubernetes secret which contains your Bitwarden credentials
-* A Docker image running the Bitwarden CLI. You could use `ghcr.io/charlesthomas/bitwarden-cli:2023.12.1` or build your own.
+* A Docker image running the Bitwarden CLI. You could use `ghcr.io/charlesthomas/bitwarden-cli:2026.3.0` or build your own.
 
 Here is an example of a Dockerfile used to build the image:
 ```dockerfile
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)